js/apps/admin-ui/src/realm-settings/EmailTab.tsx

```diff
@@ -98,301 +97,232 @@ export const RealmSettingsEmailTab = ({
+              }}
             />
-          </FormGroup>
-          <FormGroup label={t("port")} fieldId="kc-port">
-            <KeycloakTextInput
-              id="kc-port"
-              placeholder="SMTP port (defaults to 25)"
-              {...register("smtpServer.port")}
+            <TextControl
+              name="smtpServer.port"
+              label={t("port")}
+              placeholder={t("smtpPortPlaceholder")}
             />
-          </FormGroup>
-          <FormGroup label={t("encryption")} fieldId="kc-html-display-name">
-            <Controller
-              name="smtpServer.ssl"
-              control={control}
-              defaultValue="false"
-              render={({ field }) => (
-                <Checkbox
-                  id="kc-enable-ssl"
-                  data-testid="enable-ssl"
-                  label={t("enableSSL")}
-                  isChecked={field.value === "true"}
-                  onChange={(value) => field.onChange("" + value)}
-                />
-              )}
-            />
-            <Controller
-              name="smtpServer.starttls"
-              control={control}
-              defaultValue="false"
-              render={({ field }) => (
-                <Checkbox
-                  id="kc-enable-start-tls"
-                  data-testid="enable-start-tls"
-                  label={t("enableStartTLS")}
-                  isChecked={field.value === "true"}
-                  onChange={(value) => field.onChange("" + value)}
-                />
-              )}
-            />
-          </FormGroup>
-          <FormGroup
-            hasNoPaddingTop
-            label={t("authentication")}
-            fieldId="kc-authentication"
-          >
-            <Controller
+            <FormGroup label={t("encryption")} fieldId="kc-html-display-name">
+              <Controller
+                name="smtpServer.ssl"
+                control={control}
+                defaultValue="false"
+                render={({ field }) => (
+                  <Checkbox
+                    id="kc-enable-ssl"
+                    data-testid="enable-ssl"
+                    label={t("enableSSL")}
+                    isChecked={field.value === "true"}
+                    onChange={(value) => field.onChange("" + value)}
+                  />
+                )}
+              />
+              <Controller
+                name="smtpServer.starttls"
+                control={control}
+                defaultValue="false"
+                render={({ field }) => (
+                  <Checkbox
+                    id="kc-enable-start-tls"
+                    data-testid="enable-start-tls"
+                    label={t("enableStartTLS")}
+                    isChecked={field.value === "true"}
+                    onChange={(value) => field.onChange("" + value)}
+                  />
+                )}
+              />
+            </FormGroup>
+            <SwitchControl
```

Comment:
Could this be a `DefaultSwitchControl`.

Reply:
No, the labels need to be "enabled" and "disabled" and not "on" and "off"

Reply:
Ah, missed that, consider this resolved.